src/org/labkey/test/tests/SampleTypeNameExpressionTest.java

```diff
@@ -896,7 +896,7 @@ public void testNameExpressionPreview() throws IOException, CommandException
         mouseOver(createPage.getComponentElement());
 
         log("Use a name expression using a field from the named parent, with parent type not encoded.");
-        nameExpressionBad = String.format("SNP_${genId}_${%s/$s}_${materialInputs/%s/%s}", parentAlias, PARENT_FIELD_CURLY_RIGHT_INT.getExpName(), PARENT_SAMPLE_TYPE, PARENT_FIELD_CURLY_LEFT.getName());
+        nameExpressionBad = String.format("SNP_${genId}_${%s/%s}_${materialInputs/%s/%s}", parentAlias, PARENT_FIELD_CURLY_RIGHT_INT.getExpName(), PARENT_SAMPLE_TYPE, PARENT_FIELD_CURLY_LEFT.getName());
```

Comment:
This bugs me, I should have caught this when I was in this code last week. The linter was clearly calling this out but I overlooked it.